### ISSUE TITLE
Can't extract one image from remote file

### ISSUE THREAD

**Issue by xorik** [spatie/pdf-to-image#72]
```
        $url = 'http://www.axmag.com/download/pdfurl-guide.pdf';
        $pdf = new Pdf($url);
        var_dump($pdf->getNumberOfPages());
        $pdf->saveImage('1.jpg');
```
Looks like it adds `[0]` to the URL, because number of pages is working. Is it possible to workaround it? Or I need to download the file manually?

**Comment by introwit** [comment]
@xorik fixed in #76

**Comment by xorik** [comment]
@introwit the fix is incorrect. Please try this code:

```
        $url = 'https://partner.news.yandex.ru/public/documents/tech.pdf';

        $pdf = new Pdf($url);
        $pdf->setPage(1);
        $pdf->saveImage('/tmp/1.jpg');
```

You will see last page, not first
